mlrun/utils/helpers.py

```diff
@@ -1596,3 +1596,13 @@ def get_serving_spec():
             )
     spec = json.loads(data)
     return spec
+
+
+def additional_filters_warning(additional_filters, class_name):
+    if additional_filters and any(
+        additional_filter for additional_filter in additional_filters
```

Comment:
That's a redundant for expression.
```suggestion
        additional_filter
```

Reply:
what about [()]? the condition will return True, that's why I need to check every tuple in the list.

Reply:
right, tnx!